src/start_location.cpp

```diff
@@ -510,6 +510,31 @@ void start_location::handle_heli_crash( avatar &you ) const
     }
 }
 
+void start_location::wound_monster( monster *mon ) const
```

Comment:
This could be combined with handle_limb_wounds, with the "is it a monster" logic within the combined function instead of out in game.cpp where the two functions are called.